File: Code coaches/Easy/Skee-ball/SkeeBall.swift
if 	let input1 = readLine(),
		let score = Int(input1),
		let input2 = readLine(),
		let squirtGunPrice = Int(input2)
{
	let ratio = 12
	let tickets = score / ratio
	
	if tickets >= squirtGunPrice {
		print("Buy it!")
	} else {
		print("Try again")
	}
}

File: Code coaches/Easy/Skee-ball/SkeeBall.java
import java.util.Scanner;

public class SkeeBall {
	
	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		int score = input.nextInt();
		int squirtGunPrice = input.nextInt();

		int ratio = 12; // 12 points = 1 ticket
		int tickets = score / ratio;

		if (tickets >= squirtGunPrice) {
			System.out.print("Buy it!");
		} else {
			System.out.print("Try again");
		}
		
		input.close();
	}

}
